integrations/cavern-adapter/src/money_market.rs

```diff
@@ -469,7 +471,27 @@ impl Cavern {
             deps.api.addr_validate(&custody_config.collateral_token)?,
         )?;
 
-        // _env is not used here, so using mock_env
+        // mock_env() is not used inside the lsd config query exchange rate function.
+        // This is a fix because the direct query is not available on the contracts unfortunately.
+        // We can feed dummy information inside this mock_env function
         Ok(lsd_config.query_exchange_rate(deps, mock_env())?)
     }
 }
+
+/// This is only used as a fix to feed dummy information to the query_exchange_rate function
+#[cfg(feature = "full_integration")]
+fn mock_env() -> Env {
+    use cosmwasm_std::{BlockInfo, ContractInfo, Timestamp, TransactionInfo};
+
+    Env {
+        block: BlockInfo {
+            height: 12_345,
+            time: Timestamp::from_nanos(1_571_797_419_879_305_533),
+            chain_id: "cosmos-testnet-14002".to_string(),
+        },
+        transaction: Some(TransactionInfo { index: 3 }),
+        contract: ContractInfo {
+            address: Addr::unchecked("mock_addr"),
+        },
+    }
+}
```

Comment:
Small nit: can we make it minimal impact on the wasm?
```suggestion
    Env {
        block: BlockInfo {
            height: Default::default(),
            time: Default::default(),
            chain_id: Default::default(),
        },
        transaction: Default::default,
        contract: ContractInfo {
            address: Addr::unchecked(String::new()),
        },
    }
}
```
> The vector will not allocate until elements are pushed onto it.